src/NuGet.Core/NuGet.ProjectModel/PackageSpecUtility.cs

```diff
@@ -70,20 +70,31 @@ public static bool IsSnapshotVersion(string version)
             // Update the framework appropriately
             targetFrameworkInfo.FrameworkName = GetFallbackFramework(
                 targetFrameworkInfo.FrameworkName,
+                false, // Imports is null, so doesn't really matter what we have here
```

Comment:
Both of these are maintainability concerns, and I'd rather not fragment the code to have 2 overloads for the same logic.